common/stats/status_server.cpp

```diff
@@ -92,6 +98,12 @@ StatusServer::StatusServer(uint16_t port, EndPointToOPMap op_map)
       ret += itor->second(nullptr);
     }
```

Comment:
can you remove the code for rocksdb_info.txt special handling by adding "rocksdb_info.txt" to extra_stats_endpoints_?